docs/one-click-integrations/github-integration.md

```diff
@@ -44,7 +52,7 @@ Your GitHub integration is now complete. Pulse will start loading your data for
 
 The following is a detailed description of how the Pulse GitHub integration automatically detects deployment using each detection strategy:
```

Comment:
Since there are sections with titles following this paragraph, I suggest reformatting it to introduce links.

This would also require adding IDs to the sections below and tweaking the titles to fit the list format. Just an idea; we can do this further on.

```suggestion
The Pulse GitHub integration can detect deployments using the following strategies:

-   [Merged pull requests](#)  (based on default branch)
-   [Semantic versioning tags](#)
-   [CLI or API](#) (don't detect deployments automatically)
```

Reply:
Updated!